source/plugin/configuration/loaders.rst

```diff
@@ -105,18 +113,11 @@ Example: Loading a default config from the plugin jar file
 
     import java.net.URL;
 
-    URL jarConfigFile = this.getClass().getResource("defaultConfig.conf");
+    URL jarConfigFile = Sponge.getAssetManager().getAsset("defaultConfig.conf").get().getUrl();
     ConfigurationLoader<CommentedConfigurationNode> loader =
       HoconConfigurationLoader.builder().setURL(jarConfigFile).build();
 
-For this example it is important to note that the ``getResource(...)`` method works relative to the location of the
-class it is called on. So if in the above example the class lies in the package ``me.username.myplugin``, the
-``defaultConfig.conf`` file must not lie in the jar file root, but instead in the directory ``me/username/myplugin``.
-
-.. warning::
-
-    Since all Sponge plugins share a single namespace, all resources available to the ``getResource()`` method are
-    visible to all other plugins as well. Therefore, resources placed in the root of a jar may overwrite (or be
-    overwritten by) identically named resources in another jar. Placing those resources in unique folder structures
-    similar to your java packages will mitigate the danger of accidentally having a resource file overwritten by
-    another plugin.
+For this example it is important to note that the :javadoc:`AssetManager#getAsset(String)` method works relative to the
```

Comment:
@12AwsomeMan34 Do we need to import `java.lang.String` here? In the preview it links to https://jd.spongepowered.org/5.0.0/org/spongepowered/api/asset/AssetManager.html#getAsset-String- instead of https://jd.spongepowered.org/5.0.0/org/spongepowered/api/asset/AssetManager.html#getAsset-java.lang.String-